### PR TITLE
CI-492: Make it possible for administrators to change the logo in the code

### PR DESCRIPTION
**Pull request by SerhiiBorozenets**
[CI-492](https://cloverpop-internship.atlassian.net/browse/CI-492)

[CI-492]: https://cloverpop-internship.atlassian.net/browse/CI-492?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rogergraves — CHANGES_REQUESTED** [review]
@SerhiiBorozenets in order for logo changing to work on Heroku [we would need to add an additional service](https://help.heroku.com/K1PPS2WM/why-are-my-file-uploads-missing-deleted-from-the-application) to Heroku. I think this is too much trouble and ongoing costs since the logo is going to be set only once, when the project is instantiated. Please hard-code the logo into the project and change the README instructions for swapping out the logo it via code instead of ActiveAdmin.

**Review by SBorozenets — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -192,6 +194,32 @@ To address the `OpenSSL::Cipher::CipherError` error in Rails application, you ca
...
+
+### Locate the Assets Folder
+
+In your application's source code, navigate to the assets folder. This folder should contain all the visual assets used by your app, including the current logo and favicon files.
+
+### Replace the Logo
+
+Find the current logo file in the assets folder. It is usually named something like logo.svg. Remove the existing logo file from the assets folder and replace it with your new logo file, making sure to give it the same name as the original file.
+
+For example, if the original logo is logo.svg, your new logo should also be named logo.svg and replace the existing file.
```
Comment (rogergraves) [inline README.md:207]: Please replace the text here with something very specific instructions. For example:

"Save your company logo as logo.svg, and then copy over the current image in app/assets/images/logo.svg. The width to height ratio of your logo should be set to X:Y". -- please adjust to whatever the ratio is (4:3, 16:9, etc.)

File: README.md [hunk 2]
```diff
@@ -192,6 +194,32 @@ To address the `OpenSSL::Cipher::CipherError` error in Rails application, you ca
...
+
+### Replace the Favicon
+
+Similarly, find the current favicon file in the assets folder. The favicon is the small icon that appears on the browser tab. It is commonly named favicon.svg.
+
+Remove the existing favicon file from the assets folder and replace it with your new favicon file, again making sure to name it favicon.svg.
```
Comment (rogergraves) [inline README.md:213]: Suggestion:
"Copy your company's favicon (in SVG format) to  app/assets/images/favicon.svg. It should be a square image (1:1 ratio)."

File: README.md [hunk 3]
```diff
@@ -192,6 +194,32 @@ To address the `OpenSSL::Cipher::CipherError` error in Rails application, you ca
...
+
+### Save Changes
+
+Once you've replaced the logo and favicon files, save your changes to the assets folder.
+
+### Build and Deploy
+
+Depending on the technology stack of your application, you might need to build or compile your project to incorporate the new logo and favicon. Once the build is successful, deploy your application to see the updated visuals.
```
Comment (rogergraves) [inline README.md:221]: This copy is not useful and might be confusing, so please remove it.

**Review by rogergraves — APPROVED** [review]

**Pull request merged**